libraries/session-storage/impl/src/main/kotlin/io/element/android/libraries/sessionstorage/impl/SessionDataMapper.kt

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2023 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.element.android.libraries.sessionstorage.impl
+
+import io.element.android.libraries.sessionstorage.api.SessionData
+
+internal fun SessionData.toDbModel(): io.element.android.libraries.matrix.session.SessionData {
```

Comment:
If we're going to have 2 instances of classes called `SessionData` maybe it would be better to rename one of them? The `api` one could be `SessionModel` or `SessionDetails` maybe?

Reply:
Yes, it's not ideal. I wanted to avoid renaming. But since this is hidden in the impl module, I guess it's fine.